docs/UserGuide.md

```diff
@@ -78,176 +78,198 @@ Financial Assurance Revolutionary Telemarketer (FART) is a **desktop app for man
 
 ### Viewing help : `help`
 
-Shows a message explaining how to access the help page.
+Need help navigating F.A.R.T? This command opens a window that has all you need.
 
 ![help message](images/newHelp.png)
 
 Format: `help`
 
 
-### Adding a person: `add`
+### Adding a client: `add`
 
-Adds a person to the FART book.
+Met a potential client or someone new? This command adds a client to the F.A.R.T book.
 
 Format: `add n/NAME p/PHONE_NUMBER e/EMAIL [a/ADDRESS] b/BIRTHDAY [t/TAG]…​`
 
 <div markdown="span" class="alert alert-primary">:bulb: **Tip:**
-A person can have any number of tags (including 0)
+A client can have any number of tags (including 0)
 </div>
 
 Examples:
 * `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01 b/11 09 2001`
 * `add n/Betsy Crowe t/friend e/betsycrowe@example.com a/Newgate Prison p/1234567 b/11 09 2001 t/criminal`
 
 **Note:**
-1. As of version 1.4, only valid email address (in the form local-part@domain.com) and Singapore phone numbers are accepted.
+1. As of version 1.4, only valid email address (in the form name@domain.com) and **Singapore** phone numbers are accepted.
 Birthdays should be added in DD MM YYYY format.
-2. You may add duplicate contacts (i.e., contacts with the same name, case insensitive) but the AddressBook will prompt you
+2. You may add duplicate contacts (i.e., contacts with the same name, case-insensitive) but the F.A.R.T book will prompt you
 to confirm that you are adding a duplicate contact.
 
-### Listing all persons : `list`
+### Listing all clients : `list`
 
-Shows a list of all persons in the FART book.
+Need to have a quick view of everyone in your F.A.R.T book? This command displays a list of all clients in the F.A.R.T book.
 
 Format: `list`
 
```

Comment:
Believe another function of this command is to reset the list after any filtering e.g. find. Would it be good to add this point as well?